modules/Expectation.js

```diff
@@ -398,7 +399,13 @@ class Expectation {
     assert(
       typeof this.actual === 'object',
       'The "actual" argument in expect(actual).toIncludeKeys() must be an object, not %s',
-      typeof this.actual
+      inspect(this.actual)
```

Comment:
The default `assert` function will to the `inspect` for you. No need to do it here.